diamond_agent/tests/resources/blueprint/multi_node.yaml

```diff
@@ -25,7 +25,6 @@ node_templates:
                         diamond_config: { get_input: diamond_config }
                 start: diamond.diamond_agent.tasks.start
                 stop: diamond.diamond_agent.tasks.stop
-                uninstall: diamond.diamond_agent.tasks.uninstall
```

Comment:
Since in the test we check configuration files I disabled the uninstall operation which deletes those.